api/server/server.go

```diff
@@ -1058,14 +1058,11 @@ func (s *Server) postContainersWait(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	name := vars["name"]
-	cont, err := s.daemon.Get(name)
-	if err != nil {
+	status, err := s.daemon.ContainerWait(vars["name"], -1*time.Second)
+	if err != nil && !strings.Contains(strings.ToLower(err.Error()), "timed out") {
```

Comment:
But even if we did, I'm not sure we'd want to return a `200` on  a time out. That would be misleading to the user.
I think we should just remove the && part of the previous `if`